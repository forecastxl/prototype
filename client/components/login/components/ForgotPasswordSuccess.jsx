import React, { PropTypes } from 'react'

function ForgotPasswordSuccess() {
  return (
    <div>
      If your email is associated to an account with us we will send you an email with a link to reset your password. Check your email to continue.
    </div>
  )
}

ForgotPasswordSuccess.propTypes = {
  email: PropTypes.string.isRequired,
  errors: PropTypes.object.isRequired,
  onChange: PropTypes.func.isRequired,
  onSubmit: PropTypes.func.isRequired
}

export default ForgotPasswordSuccess
