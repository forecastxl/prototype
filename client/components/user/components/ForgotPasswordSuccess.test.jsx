import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ForgotPasswordSuccess from './ForgotPasswordSuccess'

describe('<ForgotPasswordSuccess />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <ForgotPasswordSuccess />
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
