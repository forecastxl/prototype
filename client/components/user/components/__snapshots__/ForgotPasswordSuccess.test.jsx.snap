// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForgotPasswordSuccess /> renders correctly 1`] = `
<div>
  If your email is associated with an account with us we will send you an email with a link to reset your password. Check your email to continue.
</div>
`;
